### ISSUE TITLE
Is the lstm_seq2seq example wrong?

### ISSUE THREAD

**Issue by sun-peach** [keras-team/keras#13267]
Hi, dear all,

I am reading the lstm_seq2seq example. However I am confused in the segment below:

`# Set up the decoder, using encoder_states as initial state.`
decoder_inputs = Input(shape=(None, num_decoder_tokens))
`# We set up our decoder to return full output sequences,`
`# and to return internal states as well. We don't use the`
`# return states in the training model, but we will use them in inference.`
decoder_lstm = LSTM(latent_dim, return_sequences=True, return_state=True)
decoder_outputs, _, _ = decoder_lstm(decoder_inputs, initial_state=encoder_states)
decoder_dense = Dense(num_decoder_tokens, activation='softmax')
decoder_outputs = decoder_dense(decoder_outputs)

Why do we have the decoder_dense a "Dense" layers? Shouldn't that be Timedistributed, since the previous layer is LSTM(return_sequences=True)?

Thank you very much!
